pkg/cmd/flagtypes/addr.go

```diff
@@ -106,6 +106,8 @@ func (a *Addr) Set(value string) error {
 
 	if !a.AllowPrefix {
 		addr.Path = ""
+	} else if addr.Path == "/" {
```

Comment:
url Parse doesn't change it, so it's only there if it's in the original string.

----- Original Message -----
> > @@ -106,6 +106,8 @@ func (a *Addr) Set(value string) error {
> >  
> >  	if !a.AllowPrefix {
> >  		addr.Path = ""
> > +	} else if addr.Path == "/" {
> 
> The behavior of url Parse is to always set this I believe.  I'm not sure we
> should be clearing it here.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2387/files#r30744095